src/devices/Ina226/Ina226Enum.cs

```diff
@@ -0,0 +1,249 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.Ina226
+{
+    /// <summary>
+    /// An enumeration representting the registers exposed by the INA226 device via I2c.
+    /// </summary>
+    internal enum Ina226Register : byte
+    {
+        /// <summary>Configuration Register r/w</summary>
+        Configuration = 0x00,
+
+        /// <summary>Shunt Voltage Register r</summary>
+        ShuntVoltage = 0x01,
+
+        /// <summary>Bus Boltage Register r</summary>
+        BusVoltage = 0x02,
+
+        /// <summary>Power Register r</summary>
+        Power = 0x03,
+
+        /// <summary>Current Register r</summary>
+        Current = 0x04,
+
+        /// <summary>Calibration Register r/w</summary>
+        Calibration = 0x05,
+
+        /// <summary>Mask/Enable Register r/w</summary>
+        AlertEnable = 0x06,
+
+        /// <summary>Alert Limit Register r/w</summary>
+        AlertLimit = 0x07,
+
+        /// <summary>Manufacturer ID Register r</summary>
+        ManufacturerID = 0xFE,
+
+        /// <summary>Die ID Register r</summary>
+        DieID = 0xFF,
+    }
+
+    /// <summary>
+    /// An enumeration representing the operating modes available on the INA226 device.
+    /// </summary>
+    public enum Ina226OperatingMode : ushort
```

Comment:
given this is in Ina226 namespace I'll recommend to remove all `Ina226` prefixes from enums (only main class makes sense)